cmd/edit/ingress/cmd.go

```diff
@@ -134,23 +208,25 @@ func run(cmd *cobra.Command, argv []string) {
 
 	clusterKey := r.GetClusterKey()
 
-	if !interactive.Enabled() && shouldEnableInteractive(cmd.Flags(), []string{labelMatchFlag, privateFlag, lbTypeFlag}) {
+	if !interactive.Enabled() && shouldEnableInteractive(cmd.Flags(),
+		[]string{labelMatchFlag, privateFlag, lbTypeFlag, routeSelectorFlag, excludedNamespacesFlag, wildcardPolicyFlag,
+			namespaceOwnershipPolicyFlag, clusterRoutesHostnameFlag, clusterRoutesTlsSecretRefFlag}) {
 		interactive.Enable()
 	}
 
 	cluster := r.FetchCluster()
 	var labelMatch *string
 	if cmd.Flags().Changed(labelMatchFlag) {
 		if ocm.IsHyperShiftCluster(cluster) {
-			r.Reporter.Errorf("Updating route selectors is not supported for hosted cp clusters")
+			r.Reporter.Errorf("Updating route selectors is not supported for Hosted Control Plane clusters")
 			os.Exit(1)
 		}
 		labelMatch = &args.labelMatch
 	} else if interactive.Enabled() && !ocm.IsHyperShiftCluster(cluster) {
 		labelMatchArg, err := interactive.GetString(interactive.Input{
-			Question: "Label match for ingress",
+			Question: "Route Selector for ingress",
 			Help:     cmd.Flags().Lookup(labelMatchFlag).Usage,
-			Default:  labelMatch,
+			Default:  args.labelMatch,
```

Comment:
NP: Little confusing to me, having the default still be `args.labelMatch` and this being Route Selector. WDYT?